lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -2873,6 +2877,20 @@ TypeSystemSwiftTypeRef::DemangleCanonicalOutermostType(
   return canonical;
 }
 
+bool TypeSystemSwiftTypeRef::IsExpressionEvaluatorDefined(
+    lldb::opaque_compiler_type_t type) {
+  using namespace swift::Demangle;
+  const auto *mangled_name = AsMangledName(type);
+  Demangler dem;
+  NodePointer node = GetDemangledType(dem, mangled_name);
+  return swift_demangle::FindIf(node, [](NodePointer node) -> NodePointer {
```

Comment:
Nit: we're silently casting from NodePointer to bool, I'd prefer if it were more explicit by comparing the result of `FindIf` with nullptr.